.github/workflows/zscaler-iac-scan.yml

```
@@ -76,6 +76,6 @@ jobs:
       #Ensure that the following step is included in order to post the scan results under the code scanning alerts section within the repository.
       - name: Upload SARIF file
         if: ${{ steps.credentials.outputs.configured == 'true' && (success() || failure()) && (steps.zscaler-iac-scan.outputs.sarif_file_path != '') }}
-        uses: github/codeql-action/upload-sarif@v3
+        uses: github/codeql-action/upload-sarif@v4
         with:
           sarif_file: ${{ steps.zscaler-iac-scan.sarif_file_path }}
```

actionlint (1.7.12)

[error] property "sarif_file_path" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)